plugins/modules/zos_find.py

```diff
@@ -238,14 +227,6 @@
     contains: 'hello'
     excludes: '.*TEST'
 
-- name: Find all members starting with characters 'TE' in a given list of PDS patterns
```

Comment:
Since the functionality is not being taken away but rather integrating it to an already existing option we should keep examples that show how to retrieve the members.